src/core/overlay/overlay-ref.ts

```diff
@@ -12,8 +12,9 @@ export class OverlayRef implements PortalHost {
       private _state: OverlayState) { }
 
   attach(portal: Portal<any>): Promise<any> {
-    return this._portalHost.attach(portal).then(() => {
+    return this._portalHost.attach(portal).then((_comp) => {
       this._updatePosition();
+      return _comp;
```

Comment:
I can't see the point of that change. 

The resolved value will be automatically returned, so there is no need to pass it as an attribute and return it "again".